index.js

```diff
@@ -14,4 +14,129 @@ cloSerX.addEventListener('click', () => {
 navListItems.forEach((item) => item.addEventListener('click', () => {
   mariMarzi.classList.add('hide');
   mariMarzi.classList.remove('mobile-menu');
-}));
\ No newline at end of file
+}));
+
+function createPopupWindow(project) {
+  const popupWindow = document.createElement('div');
+  popupWindow.classList.add('popup-window');
+  popupWindow.innerHTML = `<div>
+      <a id="close-popup"><img src="files/Icon.svg" alt="Close Button"></a>
```

Comment:
- [ ] You have done great to apply the changes required by the previous review. However, in some desktop and mobile sizes, the `close-pop` icon can not be accessed. This is because there is no margin space on top of the popup. Kindly consider applying that so that the popup can easily be closed. This will also enable us to make the title visible ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/233055984-91ee7195-1e91-4cc9-a996-57437ae19a1c.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/233056757-d247e5cf-f5a4-40ea-a522-a52adb6b4330.png)

Reply:
<img width="1554" alt="Screenshot 2023-04-19 at 4 15 50 PM" src="https://user-images.githubusercontent.com/114190100/233059316-389f0fd3-af35-4bf1-b4e7-4b5f3d26afd3.png">
   
   
   the reviewed problem is fixed